webpack/config.development.js

```diff
@@ -1,10 +1,7 @@
-'use strict';
-
-const
-    webpack = require('webpack'),
-    plugins = require('./plugins'),
-    layout = require('../layout'),
-    devServer = require('./devServer');
+var webpack = require('webpack');
```

Comment:
vars usage violates our style-guide,
Please use const where possible, and let when mutation is neseccary